### PR TITLE
Add script to create database schema

### PR DESCRIPTION
**Pull request by anabellchan**

### CODE REVIEW

File: database/mysql/seed.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const data = require('./data');
+
+const productCount = 100000;
```
Comment (jamesstockman14) [inline database/mysql/seed.js]: is this just for testing? shouldn't this be 10 million?

File: database/mysql/seed.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+for (let i = 1; i <= sellerCount; i++) {
+  seller.id = i;
+  seller.name = data.sellers[Math.floor(Math.random() * data.sellers.length)];
+  seller.policyId = data.returnPolicies[Math.ceil(Math.random() * data.returnPolicies.length)];
+  seller.deliveryId = data.delivery[Math.ceil(Math.random() * data.delivery.length)];
+}
+
+// delivery option
+
+for (let i = 1; i <= data.deliveryOptions.length; i++) {
```
Comment (jamesstockman14) [inline database/mysql/seed.js]: I'm assuming that this is just to test out seeding your db. If not I think you need to create more data than what this for loop creates.

**Review by jamesstockman14 — COMMENTED** [review]
It looks like you will need to create more instances to save into your DB. I'm assuming you just want to test with a smaller number at first which makes sense. Your code is very readable, I like the comments labeling the different parts of your code.

File: database/mysql/seed.js (hunk 1 above)
Comment (anabellchan) [reply database/mysql/seed.js]: Yes, this becomes 10 million on fake-data.js.

File: database/mysql/seed.js (hunk 2 above)
Comment (anabellchan) [reply database/mysql/seed.js]: This code is obsolete.  Updates were made to write x amount of data to csv files for mysql import.

**Pull request merged**